docs/src/docs/asciidoc/user/caching-concepts.adoc

```diff
@@ -37,41 +37,42 @@ and technical decision based upon the requirements and assumptions of your appli
 [[storage-tiers]]
 == Storage Tiers
 
-You can configure Ehcache to use various data storage areas.  When a cache is configured to use more than one data
-store, these stores are referred to as tiers.
+You can configure Ehcache to use various data storage areas. 
+When a cache is configured to use more than one storage area, those areas are arranged and managed as tiers - where
+the _hottest_ (most recently accessed) data is kept in the faster (and typically less abundant) tiers, and data that
+is less hot is confined to the slower (and more abundant) tiers.
```

Comment:
I feel this sentence is too unspecific and seems to hint at the former overflow model.I would rather have something that says:
> When a cache is configured to use more than one storage area, those areas are arranged and managed as tiers.
> The _hottest_ data is kept in the caching tier, typically less abundant.
> This tier can itself be composed of more than one storage area.
> All the data is kept in the authoritative tier which is slower but also more abundant.